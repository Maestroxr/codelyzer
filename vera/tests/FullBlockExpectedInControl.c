
int main()
{

	
	int i = 0;
	
	/*OK*/
	if (i>10)
	{
		x++;
	}		
	for (i = 0; i <10; i++) 
	{
		x++;
	}
	while (i<100) 
	{
		i++;
	}
	/*--------*/
	
	/*PROBLEM*/
	if (i>10) x++;
	for (i = 0; i <10; i++) x++;
	while (i<100) i++;
	/*********/
	
	
	return x;
}



