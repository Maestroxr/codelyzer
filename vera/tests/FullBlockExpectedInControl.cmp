C:\Work\AutoAssessment\codelyzer\vera\tests\FullBlockExpectedInControl.c:24: Full block {} expected in the control structure.
C:\Work\AutoAssessment\codelyzer\vera\tests\FullBlockExpectedInControl.c:25: Full block {} expected in the control structure.
C:\Work\AutoAssessment\codelyzer\vera\tests\FullBlockExpectedInControl.c:26: Full block {} expected in the control structure.