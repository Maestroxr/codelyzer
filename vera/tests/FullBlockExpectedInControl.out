NEXT> value:10 line:9 column:7
NEXT> value:> line:9 column:6
NEXT> value:i line:9 column:5
NEXT> value:100 line:17 column:10
NEXT> value:< line:17 column:9
NEXT> value:i line:17 column:8
NEXT> value:10 line:24 column:7
NEXT> value:> line:24 column:6
NEXT> value:i line:24 column:5
NEXT> value:100 line:26 column:10
NEXT> value:< line:26 column:9
NEXT> value:i line:26 column:8
C:\Work\AutoAssessment\codelyzer\vera\tests\FullBlockExpectedInControl.c:24: Full block {} expected in the control structure.
C:\Work\AutoAssessment\codelyzer\vera\tests\FullBlockExpectedInControl.c:25: Full block {} expected in the control structure.
C:\Work\AutoAssessment\codelyzer\vera\tests\FullBlockExpectedInControl.c:26: Full block {} expected in the control structure.
